cips/cip-14.md

```diff
@@ -213,7 +213,18 @@ This proposal is backwards-incompatible because it is state-machine breaking. Th
 
 ## Test Cases
 
-TBA
+The following test cases are available in the [ibc-go e2e repository.](https://github.com/cosmos/ibc-go/tree/main/e2e/tests/interchain_accounts)
+
+- Registration of an interchain account
+- Transfer funds from interchain account to a different account on the same chain using an unordered or ordered channel
+- A failed transfer of funds from interchain account to a different account on the same chain due to insufficient funds
+- Transfer funds from interchain account to a different account on the same chain after an ordered channel closure
+- A transfer of funds from an interchain account to a different account on the same chain using a gov controller
+- A transfer of funds from an interchain account to a different account on the same chain using a groups controller
+- A transfer of funds from an interchain account to a different account on the same chain using an incentivised IBC packet
```

Comment:
An incentivized IBC packet would make this CIP depend on https://github.com/celestiaorg/CIPs/blob/main/cips/cip-12.md

Since this CIP doesn't claim to depend on CIP in the header [here](https://github.com/celestiaorg/CIPs/pull/99/files#diff-068ed9a53bb7c0265a6380560ea0bd0e80d14779a3d9366492ec02a080ca1d08R1-R11) can this test case be listed as optional